samcli/local/init/templates/cookiecutter-aws-sam-hello-dotnet/{{cookiecutter.project_name}}/Solution.sln

```diff
@@ -0,0 +1,43 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Should this have been committed? Looks like a ide generated file.

Reply:
@jfuss in dotnet world the Solution file "glues" the projects. It is an artifact used to make it easy to restore NuGet packages or build the projects in one go. It is an abstraction created by Microsoft.
Started with Visual Studio, but if you use JetBrains Rider, it recognises and uses it.

When you are using Visual Studio Code you do not need it, since VS Code opens a folder.

Reply:
No problem. Sharing is caring! ;)